spec/msk_cluster_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'vcr_helper'
+
+ACTUAL_MSK_ARN = 'arn:aws:kafka:us-east-2:373045849756:cluster/rjr-test-quasar/430c3363-00b0-4576-ae82-6894eb9ea401-3'
+RSpec.describe OpenStax::Aws::MskCluster, vcr: VCR_OPTS do
+  let(:instance) do
+    described_class.new(
+      cluster_arn: ACTUAL_MSK_ARN, region: 'us-east-2'
+    )
+  end
+
+  it 'Retrieves list of bootstrap hosts' do
```

Comment:
minor comment.  it descriptions shouldn't be capitalized...it should read as a sentence.   https://www.betterspecs.org/

Reply:
thanks, will note for future

Reply:
changed. (and improved. And moved the dang cassette, that uses the description as a filename. Oops)